tests/mir-opt/unnamed-fields/field_access.rs

```diff
@@ -36,18 +37,36 @@ union Bar {
 
 fn access<T>(_: T) {}
 
+// CHECK-LABEL: fn foo(
 fn foo(foo: Foo) {
+    // CHECK _3 = (_1.0: u8);
+    // CHECK _2 = access::<u8>(move _3) -> [return: bb1, unwind: bb5];
```

Comment:
iirc filecheck supports backreferences. Can you try making the `_3` a variable that then gets used in the `move _3`? If it doesn't look easy, just comment that here and we'll land it as is.

Reply:
I have assigned variable names for `_3` and `_2` mir output variables.